src/app/zeko/circuits/rule_commit.ml

```diff
@@ -177,7 +177,9 @@ struct
        max_valid_while_size must be at least 1. *)
     let* () =
       assert_var __LOC__ (fun () ->
-          let* diff = Slot.Checked.diff slot_range.upper slot_range.lower in
+          (* FIXME: This is just for testing, remove later *)
```

Comment:
Again, we shouldn't commit stuff like this to `compatible`. Keep it in a branch.